src/pages/patch/patchTabs/tasks/TasksTable.tsx

```diff
@@ -62,6 +64,65 @@ const getSortDirFromOrder = (order: "ascend" | "descend") =>
 
 const rowKey = ({ id }: { id: string }): string => id;
 
+const mapTaskStatusToBadgeVariant = {
+  [TaskStatus.Inactive]: Variant.LightGray,
+  [TaskStatus.Unstarted]: Variant.LightGray,
+  [TaskStatus.Undispatched]: Variant.LightGray,
+  [TaskStatus.Started]: Variant.Yellow,
+  [TaskStatus.Dispatched]: Variant.LightGray,
+  [TaskStatus.Succeeded]: Variant.Green,
+  [TaskStatus.Failed]: Variant.Red,
+  [TaskStatus.StatusBlocked]: Variant.DarkGray,
+  [TaskStatus.StatusPending]: Variant.Yellow
```

Comment:
Pending should be the same grey that unstarted is

Reply:
ty!